python/paddle/fluid/tests/unittests/test_jit_save_load.py

```diff
@@ -863,6 +864,94 @@ def test_jit_save_load_multi_methods_inputspec(self):
                 layer, model_path, input_spec=[InputSpec(shape=[None, 784])])
 
 
+class LayerSaved(fluid.dygraph.Layer):
```

Comment:
For our 2.0 API, could you inherit from "paddle.nn.Layer" instead of "fluid.dyraph.Layer"?

Reply:
Done,thx.